src/main/java/com/library/constant/MessageType.java

```diff
@@ -0,0 +1,5 @@
+package com.library.constant;
+
+public enum MessageType {
```

Comment:
For what reason this enum is used?
Does it help in code or make code more complicated?

Reply:
It help me in RabbitProducer.class to take out common logic to private method